includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1665,26 +1665,33 @@ protected function build_display_items( $itemized_display_items = false ) {
 			define( 'WOOCOMMERCE_CART', true );
 		}
 
-		$items     = [];
-		$subtotal  = 0;
-		$discounts = 0;
+		$items         = [];
+		$lines         = [];
+		$subtotal      = 0;
+		$discounts     = 0;
+		$display_items = ! apply_filters( 'wc_stripe_payment_request_hide_itemization', true ) || $itemized_display_items;
 
-		// Default show only subtotal instead of itemization.
-		if ( ! apply_filters( 'wc_stripe_payment_request_hide_itemization', true ) || $itemized_display_items ) {
-			foreach ( WC()->cart->get_cart() as $cart_item_key => $cart_item ) {
-				$amount         = $cart_item['line_subtotal'];
-				$subtotal      += $cart_item['line_subtotal'];
-				$quantity_label = 1 < $cart_item['quantity'] ? ' (x' . $cart_item['quantity'] . ')' : '';
-
-				$product_name = $cart_item['data']->get_name();
+		foreach ( WC()->cart->get_cart() as $cart_item_key => $cart_item ) {
+			$subtotal      += $cart_item['line_subtotal'];
+			$amount         = $cart_item['line_subtotal'];
+			$quantity_label = 1 < $cart_item['quantity'] ? ' (x' . $cart_item['quantity'] . ')' : '';
+			$product_name   = $cart_item['data']->get_name();
+
+			$lines[] = [
+				'label'  => $product_name . $quantity_label,
+				'amount' => WC_Stripe_Helper::get_stripe_amount( $amount ),
+			];
```

Comment:
@dechov Sorry! I went ahead and merged the PR without looking to this comment. since the suggested change is small I hope there's no problem that I've merged the PR without addressing this comment.

Reply:
Definitely no problem – if you especially agree with the change then it would be appropriate to open another PR and I can give a quick review, but otherwise it's all done 👍